README.md

```diff
@@ -4,35 +4,32 @@
+Browse all current and draft EIPs on [the official EIP site](https://eips.ethereum.org/).
 
 Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft EIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your EIP contains either your GitHub username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
 
-When you believe your EIP is mature and ready to progress past the draft phase, you should do one of two things:
+## Project Goal
 
- - **For a Standards Track EIP of type Core**, ask to have your issue added to [the agenda of an upcoming All Core Devs meeting](https://github.com/ethereum/pm/issues), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the EIP editors will update the state of your EIP to 'Accepted'.
- - **For all other EIPs**, open a PR changing the state of your EIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the EIP - they may close the PR and request that you fix the issues in the draft before trying again.
+The Ethereum Improvement Proposals repository exists as a place to share concrete proposals with potential users of the proposal and the Ethereum community at large.
 
-# EIP Status Terms
+We promote high-quality, peer-reviewed proposals in two categories:
 
-* **Draft** - an EIP that is undergoing rapid iteration and changes.
-* **Last Call** - an EIP that is done with its initial iteration and ready for review by a wide audience.
-* **Accepted** - a core EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Core Devs to decide whether to encode an EIP into their clients as part of a hard fork is not part of the EIP process. If such a decision is made, the EIP will move to final.
-* **Final (non-Core)** - an EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - an EIP that the Core Devs have decided to implement and release in a future hard fork or has already been released in a hard fork. 
+- [Consensus Changes](EIPS/eip-1.md#eip-types) require people running an Ethereum client to update their software to implement the change. If enough people do not upgrade then an existing network will fork into two separate networks.
+- [Other Proposals](EIPS/eip-1.md#eip-types) do not require a change in client software and can be used by any interested party.
```

Comment:
What value do we get in making this distinction?  Why is this distinction more valuable than the current Core, Network, Interface, ERC?

Reply:
I believe the audience for README.md will be the wide community of Ethereum users. This includes people that will be coming by (driving by?) to make their opinions heard on the likes of the next contentious hard fork, as well as others that are implementers/developers.

I'm guessing that anybody who has more than one Ether for more than one or two years will have a strong opinion on The DAO fork, and maybe even the latest fork. But I'll bet that less than 50% would know if those were Core, Network, Interface, or ERC EIPs.

Therefore I have added this distinction in plain language so that the intended audience can easily understand what they are looking at.

Reply:
I suspect that the number of people in the intersection of "drive by brigadier" and "stopped to read the Readme" and "didn't comprehend the categories" is small enough that it isn't worth adding complexity (another layer of abstraction) to the EIP process, EIPs page, etc.

Reply:
We disagree on the target audience.

But we agree that this PR can go on without that part.

Reply:
If you want to extract these two lines out into a separate PR we can merge this without being blocked by this discussion, and then we can have a more focused discussion on just this topic (with hopefully more people getting involved).  I'm 👍 to merge everything else, but would like to discuss these lines in particular more before merging.

Reply:
Note: I'm OK with the deletion above these two lines being part of this PR, even if these two lines aren't included in this PR.  We would want to remove line 27 (of your branch) as well and pull it out to the new PR if we decide to go that route.  So the new PR would only add the following, and this PR would remove the status description section (which really should be in EIP-1 only anyway).
> We promote high-quality, peer-reviewed proposals in two categories:
> - [Consensus Changes](EIPS/eip-1.md#eip-types) require people running an Ethereum client to update their software to implement the change. If enough people do not upgrade then an existing network will fork into two separate networks.
> - [Other Proposals](EIPS/eip-1.md#eip-types) do not require a change in client software and can be used by any interested party.

Reply:
Fixed at 922a1abe861dde67abf4eda9486e380be49c9968